tests/test_color_eager.py

```diff
@@ -146,12 +156,16 @@
         "yuv_to_rgb",
         "rgb_to_xyz",
         "xyz_to_rgb",
+        "rgb_to_lab",
+        "lab_to_rgb",
         "rgb_to_grayscale",
     ],
 )
 def test_color(data, func, check):
     """test_io_color"""
 
+    np.random.seed(1000)
```

Comment:
Is this only needed for this particular test?

Reply:
@terrytangyuan the setup in the above all takes random numbers:
```
...
        pytest.param(
             lambda: (np.random.random((10, 20, 3))).astype(np.float32),
             tfio.experimental.color.rgb_to_lab,
             skimage.color.rgb2lab,
         ),
...
```

So use random seed to make sure the output is predicable.